pam/integration-tests/vhs-helpers_test.go

```diff
@@ -563,15 +563,21 @@ func evaluateTapeVariables(t *testing.T, tapeString string, td tapeData, testTyp
 	}
 
 	waitForPromptText := func(matches []string, style pam.Style) {
-		fullMatch, prefix, promptValue := matches[0], matches[1], matches[2]
+		fullMatch, prefix, context, promptValue := matches[0], matches[1], matches[3], matches[4]
```

Comment:
In Go code, `context` usually refers to the `context.Context` type. Maybe we could use `promptType` or something similar here?

Reply:
Ah, I see. Yes, both of your suggestions would be fine for me.